task/buildah/0.2/buildah.yaml

```diff
@@ -107,6 +107,10 @@ spec:
     description: Additional key=value labels that should be applied to the image
     type: array
     default: []
+  - default: "false"
+    description: Whether to use privileged nested containers when run remotely, this will add --privileged to podman run and --security-opt=label=disable --cap-add=all --device=/dev/fuse to buildah build
+    name: PRIVILEGED_NESTED
```

Comment:
> when run remotely, this will add --privileged to podman

Does this option only work for the remote variants of the task? If so, we should either call that out more clearly or only add the option to the remote tasks

Reply:
Updated the description